.codecov.yml

```diff
@@ -26,6 +26,9 @@ coverage:
   range: 100..100
   status:
     # require patches to be 100%
```

Comment:
This comment is not related to “project” but to “patch” so it should be moved. Such cases are why I prefer code comments on the same line and not separate, whenever possible...

Reply:
Oh yeah I was going to get back to this once I've got 100% reliably